packages/dashmate/src/listr/tasks/platform/featureFlagTaskFactory.js

```diff
@@ -35,6 +35,8 @@ function featureFlagTaskFactory() {
             },
           });
 
+          await ctx.client.platform.initialize();
```

Comment:
It's supposed to be initialized automatically when we call `ctx.client.platform.contracts.get`